feil-behandler/src/main/resources/db/migration/V2___bakgrunnsjobb.sql

```diff
@@ -0,0 +1,14 @@
+create table bakgrunnsjobb
+(
+    jobb_id      uuid unique  not null,
+    type         VARCHAR(100) not null,
+    behandlet    timestamp,
+    opprettet    timestamp    not null,
+
+    status       VARCHAR(50)  not null,
```

Comment:
varchar -> text

Reply:
Kan gjøre konverteringen som et delta-script i V3 for å unngå migreringsfeil. Må bare sjekke at bgjobb-koden ikke har noen issues med text vs varchar først, men det bør den ikke

Reply:
Kan du ikke bare flushe databasen?